sdks/nodejs/package.json

```diff
@@ -4,13 +4,13 @@
         "url": "https://github.com/googleforgames/agones/issues"
     },
     "dependencies": {
-        "@grpc/grpc-js": "1.7.3",
+        "@grpc/grpc-js": "1.10.7",
```

Comment:
Updating dependencies and transitive dependencies to keep ahead of security issues